lib/pbench/test/unit/server/test_endpoint_configure.py

```diff
@@ -17,27 +17,42 @@ def test_query(self, client, server_config):
         local Flask mocked environment, `request.host` will always be
         "localhost".
         """
-        self.check_config(client, server_config, "localhost")
+        self.check_config(
```

Comment:
I think we should still test what happens *without* the header.

Reply:
Well, yeah...once we decide what is supposed to happen in that case....

Reply:
That's right and if we do `proto = request.headers.get("X-Forwarded-Proto", "https")` then the default is gonna be `https`.